app/views/comments/_comments.html.slim

```diff
@@ -0,0 +1,10 @@
+- content_for :javascript do
```

Comment:
I thought it will be good idea to include comment.js only in page where we using commenting feature.
What do you mean? What js to initialize?